<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibracion</class>
 <widget class="QMainWindow" name="Calibracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibracion</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="TabControl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab1Conectar">
     <attribute name="title">
      <string>Conectar</string>
     </attribute>
     <widget class="QPushButton" name="BotonConectar">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Conectar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ComboPuertosSeries">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Puerto Serie</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>-120</y>
        <width>900</width>
        <height>900</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="whatsThis">
       <string notr="true"/>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="accessibleDescription">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>9600</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Baudios:</string>
      </property>
     </widget>
     <zorder>graphicsView</zorder>
     <zorder>BotonConectar</zorder>
     <zorder>ComboPuertosSeries</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
    </widget>
    <widget class="QWidget" name="TabCalibracion">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Calibracion</string>
     </attribute>
     <widget class="QwtPlot" name="Plot1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>611</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QwtTextLabel" name="Resutlado">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>440</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QwtTextLabel" name="TextLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>470</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CalADC">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Calibraci√≥n de sensores</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>341</width>
        <height>192</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <column>
       <property name="text">
        <string>Posicion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> Medida ADC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sigma</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Fila</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="GenerarCodigo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>590</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar Codigo</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>500</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1000</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de muestras para realizar la medida</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="CodigoText">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>641</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ComboSensores">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>120</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Sensor0 PE3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor1 PE2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor2 PE1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor3 PE0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor4 PD3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor5 PD2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor6 PD1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor7 PD0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor8 PE5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor9 PE4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor10 PB4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sensor11 PB5</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>100</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sensor</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
